src/components/GameControls/GameControls.jsx

```diff
@@ -0,0 +1,11 @@
+const GameControlls = () => {
+  return (
+    <div>
+      GameControlls
+      <button>Battle</button>
+      <button>Battle</button>
```

Comment:
- [ ] I think we need a `battle button` &  a `stop button`. Here I see 2 battle buttons. Do u plan to edit this further 👍🏾

Reply:
We need a button, where text changes from 'Buttle' to 'Stop'  and reverse